resources/main.css

```diff
@@ -19,17 +19,14 @@ code {
     font-size: smaller;
 }
 
-button {
-    cursor: pointer;
-}
-
 hr {
     border: 1px solid rgb(235, 235, 235);
     width: 50%;
     margin: 40px auto;
 }
 
 img {
+    user-select: none;
     -webkit-user-select: none;
```

Comment:
`user-select` seems to not work as expected in Safari 16.0, thus I left this in.